### DISCUSSION TITLE
[Feature Request]: 几个小建议

### DISCUSSION THREAD

**Discussion by 4Nest** [jxxghp/MoviePilot#3365] [讨论]
### 当前程序版本

v1.1.8

### 功能改进

这两天从NT转到MP，稍高强度的使用总结下来的几个小建议。
还有几个小功能的需求不知可否实现。

1、**历史记录**界面归类排序
原因：当大规模整理的时候，如果出现了一些失败的。可能会因为成功记录刷新到后面去，这样会导致处理失败记录的时候要去后面找，比较耗时耗力。
建议：增加**状态**的排序功能，能够排序出失败的记录
![image](https://github.com/jxxghp/MoviePilot/assets/74977868/0a0d4969-4eda-49d6-a18b-5428578bb85e)

2、**订阅界面UI**优化
原因：当订阅的剧比较多的时候，同时显示**标题**、**简介**等信息。占满整个界面的时候，文字在页面权重过重，在查找某部剧的订阅情况时比较费眼睛。
建议：保留**标题**信息，隐藏**简介**等信息。当hover时显示别的信息。适当简化界面，放松眼睛。
![image](https://github.com/jxxghp/MoviePilot/assets/74977868/1b2c26df-a9de-46a1-a93f-76e99defca31)

3、**自定义订阅**
提议功能：
1、增加一个TMDB ID选项，精准定位。防止同名影片撞车，目前使用bug可能是#538 导致 。
2、增加一个**仅下载**选项，这样可以同时订阅两个源的影片。如果加入同资源覆盖功能后，此功能可以实现简单的洗版覆盖或同资源不同版本选择。（个人用于动漫字幕组洗版，某站web资源速度快先追，后洗版字幕组）
3、增加**开始集数**功能，用于长连载剧的追新，不下载旧资源。

4、**删除下载记录**
原因：当有时候删除订阅后重新订阅，会显示已经下载过该种子，不会重新下种。导致卡订阅
建议：当删除订阅信息时联动删除下载记录。
（这个只是我的简单分析提议，毕竟我也不清楚流程，仅反馈现象和建议）

**Comment by dlovew** [comment]
3、自定义订阅
提议功能：

2、增加一个仅下载选项，这样可以同时订阅两个源的影片。如果加入同资源覆盖功能后，此功能可以实现简单的洗版覆盖或同资源不同版本选择。（个人用于动漫字幕组洗版，某站web资源速度快先追，后洗版字幕组）
这个问题可以通过修改命名规则，不同分辨率或者制作组可以分别保留
